### PR TITLE
10.9 Release notes

### PR DESCRIPTION
**Pull request by lzdanski**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -148,6 +148,26 @@ Refer to the [Airflow release notes](https://airflow.apache.org/docs/apache-airf
 - Fixed a bug where Airflow would show `failed_upstream` when a dynamically mapped task was `skipped`.
 - In the Python task decorator, you can only have `None` as the default parameter for context parameters.
 
+## Astro Runtime 10.9.0
+
+- Release date: June 12, 2024
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:153]:
```suggestion
- Release date: June 13, 2024
```

**Review by utkarsharma2 — APPROVED** [review]
LGTM

**Comment by ephraimbuddy** [comment]
A new CVE was just announced. Can we add it to this release:

```
### Security fixes

- [CVE-2024-25142](https://www.cve.org/CVERecord?id=CVE-2024-25142)
```

File: astro/runtime-provider-reference.md [hunk 2]
```diff
@@ -359,6 +359,32 @@ docker run --rm quay.io/astronomer/astro-runtime:11.0.0 pip freeze | grep apache
 </TabItem>
 </Tabs>
 
+## Astro Runtime 10.9.0
+
+| Package Name                             | Version |
+| :--------------------------------------- | :------ |
+| apache-airflow-providers-amazon  |  8.24.0 |
+| apache-airflow-providers-celery  |  3.7.2 |
+| | apache-airflow-providers-cncf-kubernetes  |  7.14.0 |
```
Comment (lzdanski) [inline astro/runtime-provider-reference.md:368]:
```suggestion
| apache-airflow-providers-cncf-kubernetes  |  7.14.0 |
```

**Comment by lzdanski** [comment]
- reviewed by airflow team and confirmed build in preview

**Pull request merged**